packages/@aws-cdk/aws-eks/lib/helm-chart.ts

```diff
@@ -80,9 +87,10 @@ export class HelmChart extends Construct {
       properties: {
         ClusterName: props.cluster.clusterName,
         RoleArn: props.cluster._getKubectlCreationRoleArn(provider.role),
-        Release: props.release || this.node.uniqueId.slice(-63).toLowerCase(), // Helm has a 63 character limit for the name
+        Release: props.release || this.node.uniqueId.slice(-53).toLowerCase(), // Helm has a 53 character limit for the name
         Chart: props.chart,
         Version: props.version,
+        Wait: props.wait || false,
```

Comment:
```suggestion
        Wait: props.wait === 'undefined' ? false : props.wait,
```

Reply:
@eladb actually, I should have probably looked at this more closely before committing it. It produces this error:

`lib/helm-chart.ts:93:15 - error TS2367: This condition will always return 'false' since the types 'boolean | undefined' and 'string' have no overlap.`

What's wrong with `props.wait || false` ? I just added another test to verify that it defaults to `false` if not specified, and it works as intended as far as I can tell.